skeletor/export/js.config.js

```diff
@@ -10,12 +10,21 @@ module.exports = {
 				input,
 				output: output(outputDir),
 				plugins: [
-					plugins(),
+					...plugins(),
 					require('rollup-plugin-terser').terser()
 				],
 				experimentalCodeSplitting: true
 			}
 		},
+		{
+            name: '@deg-skeletor/plugin-rollup',
+            config: {
+                input: input, 
+                output: output(`export/js`, false),  
+                plugins: plugins(false),
+                experimentalCodeSplitting: true
```

Comment:
[fix] Can you add terser to the plugins (like on line 14) and replace `export/js` with your `outputDir` variable?

Reply:
yep done.